autopep8.py

```diff
@@ -3197,6 +3257,13 @@ def create_parser():
                         help='only fix errors found within this inclusive '
                              'range of line numbers (e.g. 1 99); '
                              'line numbers are indexed at 1')
+
+    parser.add_argument('--line-ranges', metavar='ranges',
+                        default=None, type=str, nargs=1,
+                        help='only fix errors found within this inclusive '
+                        'ranges of line numbers, use dashes within ranges '
+                        'and commas between ranges. E.g: 4-8,15,16-23,42 '
+                        'line numbers are indexed at 1')
```

Comment:
Could you add to code  in `parse_args()` method, that mutually exclusive control of `--line-range` option. (same as `--diff` and `--in-place`)